docs/indepth/tutorial/Vagrantfile.template

```diff
@@ -12,6 +12,7 @@ ENV['VAGRANT_DEFAULT_PROVIDER'] = 'virtualbox'
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = "clusterhq/flocker-tutorial"
+  config.vm.box_url = "https://clusterhq-archive-testing.s3.amazonaws.com/vagrant/flocker-tutorial.json"
   config.vm.box_version = "= |latest-installable|"
```

Comment:
This is an existing bug I just thought of - `|latest-installable|` is not normalised using our `vagrant_version` - that just happens to be for now because our versions have been the same as their normalised counterparts. I'll create a follow-up.